RabbitMQ.Stream.Client/RoutingClient.cs

```diff
@@ -180,8 +186,14 @@ await LookupConnection(clientParameters, metaDataInfo.Leader, MaxAttempts(metaDa
         public static async Task<IClient> LookupRandomConnection(ClientParameters clientParameters,
```

Comment:
@Gsantomaggio could it be a good idea to rename this method? We are no longer looking up for any "random" connection, but for a random "replica" connection (if any) or a "leader" connection... is that significant for the callers of this method?

Reply:
done via https://github.com/rabbitmq/rabbitmq-stream-dotnet-client/pull/352/commits/cfb6b241d4007a65d9ebf84d717f6afddfdd5b76